warehouse/upload.go

```diff
@@ -1292,6 +1292,7 @@ func (job *UploadJobT) triggerUploadNow() (err error) {
 		metadata = make(map[string]string)
 	}
 	metadata["nextRetryTime"] = time.Now().Add(-time.Hour * 1).Format(time.RFC3339)
```

Comment:
also set a custom priority - something like 50 so that triggered uploads are picked up faster
We can even take this as an input if required

Reply:
We are already handling this when are are doing initUpload.